examples/image_classifier/resnet_152_batch/resnet152_handler.py

```diff
@@ -97,7 +96,8 @@ def preprocess(self, request):
         return image_tensor
 
     def inference(self, img):
-        return self.model.forward(img)
+        with torch.no_grad():
+            return self.model.forward(img)
```

Comment:
@dhaniram-kshirsagar Will this work properly for both cpu/gpu? Device has not been set for the inputs for this batch inference case.

Reply:
@chauhang This has been taken care in PR #432 [under review]. Per my last comment closing this one to avoid further confusion.